packages/react/src/OrderedList/index.ts

```diff
@@ -1,2 +1,3 @@
 export { OrderedList } from './OrderedList'
 export type { OrderedListProps } from './OrderedList'
+export type { OrderedListItemProps } from './OrderedListItem'
```

Comment:
What is the reason for exporting the ItemProps but not the component? The component is recognized by your IDE anyway so now there are different paths. This may look weird when a developer is extending the item for some reason.

Reply:
Yeah, good catch, I'm not sure we're exporting the right types for the top component. I've made [a new ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-591) where we can figure that out